config/routes.rb

```diff
@@ -43,7 +43,7 @@
     scope "/pages" do
       get "/" => "page_list#edit", as: :form_pages
       post "/" => "page_list#update"
-      post "/move-page" => "page_list#move_page", as: :move_page
+      post "/move-page" => "pages#move_page", as: :move_page
```

Comment:
It's not obvious to me why we're making this page, would you mind adding more detail to the commit message?

Reply:
759376e825804e37cc382871ffd5e74e89c1d815